tools/gulptasks/dashboards/scripts-dts/dashboards.src.d.ts

```diff
@@ -33,11 +33,9 @@ export { default as DataModifier } from "./es-modules/Data/Modifiers/DataModifie
 export { default as DataPool } from "./es-modules/Data/DataPool";
 export { default as DataTable } from "./es-modules/Data/DataTable";
 
-export const addEvent: typeof Utilities.addEvent;
```

Comment:
Don't you need it anymore?

Reply:
Correct, but it broke at that moment. These need to be copied over as mentioned in chat and then it can be restored.